cloud/k8s/tools/hack_node.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Kontain
+# Derived from: 
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kontain-node-hack
+  namespace: kube-system
```

Comment:
Maybe use a different namespace for this like 'default' else the developer might forget to clean up the pod in kube-system thus affecting the quota for the namespace.

Reply:
If you leave namespace blank it defaults to the current context.